release.config.js

```diff
@@ -170,6 +170,8 @@ addPlugin("@semantic-release/git", {
     "CHANGELOG.md",
     "package.json",
     "package-lock.json",
+    "**/package.json",
```

Comment:
To include all package managers this should work:

```
"package.json",
"package-lock.json",
"npm-shrinkwrap.json",
"yarn.lock",
"pnpm-lock.yaml",
"**/package.json",
"**/package-lock.json",
"**/npm-shrinkwrap.json",
"**/yarn.lock",
"**/pnpm-lock.yaml",
```

Reply:
https://github.com/open-sauced/release/commit/73b8c0d70db730b91a4b7c99ab0ad88ab2bdc9c2